src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import RenderInputTextField from '../RenderInputTextField';
+import FieldLabel from '../FieldLabel';
+import RichEditor from '../RichEditor';
+
+function AdditionalMetadataFields(props) {
+  const {
+    disabled,
+  } = props;
+
+  return (
+    <div className="collapsible-card pgn_collapsible mt-4">
+      <div className="collapsible-body">
+        <FieldLabel text="Executive Education details" className="h3 font-weight-normal mb-3" />
```

Comment:
Just a comment that  this model will be used for both Executive Education and Bootcamps once the bootcamp data fields are confirmed.

Reply:
Then we can Label it as `Additional Details`. What do you suggest?

Reply:
Yes, makes sense.